qiita_db/user.py

```diff
@@ -360,8 +371,8 @@ def shared_studies(self):
     @property
     def private_analyses(self):
         """Returns a list of private analysis ids owned by the user"""
-        sql = ("Select analysis_id from qiita.analysis WHERE email = %s AND "
-               "analysis_status_id <> 6")
+        sql = ("Select analysis_id from qiita.analysis "
```

Comment:
Same comments as above. Also the capitalization of this SQL statement is weird (at the beginning I did not notice that it was SQL...)